swarm/storage/common_test.go

```diff
@@ -34,7 +34,7 @@ import (
 )
 
 var (
-	loglevel = flag.Int("loglevel", 2, "verbosity of logs")
+	loglevel = flag.Int("loglevel", 3, "verbosity of logs")
```

Comment:
Minor thing, but I prefer the loglevel to be a command line parameter than a hardcoded value

Reply:
This is what a `flag` is - it is a cmdline parameter.

Reply:
Oh sorry, I did not read it well :)